ios/RNLinksdk.m

```diff
@@ -237,7 +235,15 @@ - (void)stopObserving {
         self.exitCallback = onExit;
         self.presentingViewController = RCTPresentedViewController();
         NSDictionary *options = self.institutionID.length > 0 ? @{@"institution_id": self.institutionID} : @{};
-        [self.linkHandler openWithContextViewController:self.presentingViewController options:options];
+
+        __weak typeof(self) weakSelf = self;
+        void(^presentationHandler)(UIViewController *) = ^(UIViewController *linkViewController) {
+            [weakSelf.presentingViewController presentViewController:linkViewController animated:true completion:nil];
+        };
+        void(^dismissalHandler)(UIViewController *) = ^(UIViewController *linkViewController) {
+            [weakSelf dismiss];
```

Comment:
If this is an intentional code change I think the `dismissLinkViewController` method implementation can be removed. If it isn't I think `dismissLinkViewController` should be called here…
```suggestion
            [weakSelf dismissLinkViewController];
```

Reply:
`dismiss` is the exported API exposed by the RN layer, it exits Link and cleans up any resources (i.e. the handler and reference to the `presentedViewController`), it calls `dismissLinkViewController` in an attempt to be more readable.

On second thought though, `dismissLinkViewController` should probably never be called outside of `dismiss`, so I'll inline it to avoid the confusion